requirements.txt

```diff
@@ -6,3 +6,4 @@ numpy==1.24.3
 pandas==2.0.1
 scikit-learn==1.2.2
 seaborn==0.12.2
+XGBOOST

```

Comment:
provide the version of xgboost.

Reply:
Can you mention the issue number please?

Reply:
45